packages/hello-gsm-calculate/src/middleware.ts

```diff
@@ -0,0 +1,10 @@
+import { NextRequest, NextResponse, userAgent } from 'next/server';
+
+export function middleware(req: NextRequest) {
+  const { origin, pathname } = req.nextUrl;
+  const { browser } = userAgent(req);
+
+  if (browser.name === 'IE' && pathname === '/') {
```

Comment:
```suggestion
  if (browser.name === 'IE' && pathname !== '/browser') {
```

위처럼 하면 IE 브라우저에서 `/browser` 를 제외하고 모든 곳에서 `/browser`로 리다이렉트 시킬 수 있을 것 같은데 어떤가요?

Reply:
오오 저건 생각을 못했네요 좋은 것 같아요 !

Reply:
> 오오 저건 생각을 못했네요 좋은 것 같아요 !

hello-gsm workspace에도 같은 상태라 고려해보셔도 좋을 것 같아요!

Reply:
https://hello-br0o2y1qm-hello-gsm.vercel.app/ 
혹시 테스트 가능하신가요?

Reply:
내 IE가 어디있더라..